src/playwright/playwright-plugin.ts

```diff
@@ -1,31 +1,21 @@
 import { Browser as PlaywrightBrowser, BrowserType } from 'playwright';
+import { Browser as PlaywrightBrowserWithPersistentContext } from './browser';
+import { PlaywrightController } from './playwright-controller';
 import { BrowserController } from '../abstract-classes/browser-controller';
 import { BrowserPlugin } from '../abstract-classes/browser-plugin';
 import { LaunchContext } from '../launch-context';
-import { noop } from '../utils';
-import { Browser as ApifyBrowser } from './browser';
-import { PlaywrightController } from './playwright-controller';
-
-export type PlaywrightPluginBrowsers = ApifyBrowser | PlaywrightBrowser;
 
-/**
- * Playwright
- */
-export class PlaywrightPlugin extends BrowserPlugin<BrowserType, Parameters<BrowserType['launch']>[0], PlaywrightPluginBrowsers> {
-    private _browserVersion!: string;
+export class PlaywrightPlugin extends BrowserPlugin<BrowserType, Parameters<BrowserType['launch']>[0], PlaywrightBrowser> {
+    declare _browserVersion: string;
```

Comment:
Can you elaborate? I don't know what's the question.

Reply:
If you mean that it should be `private`, fixed.

Reply:
That..but what's the point of this declare exactly, considering it doesn't exist.. is it related to that proxy or? Could we get a simple comment explaining why it's there? 🙏

Reply:
You're right, `declare` is moot here. It can be written as:

```ts
private _browserVersion?: string;
```

which works flawlessly as there's this check:

https://github.com/apify/browser-pool/blob/93cdf5557fd1455480074be7c3395355f931be85/src/playwright/playwright-plugin.ts#L25-L33